source/ui.js

```diff
@@ -120,11 +120,29 @@ const checkNewFilesAndDependencies = async (pkg, rootDir) => {
 };
 
 // eslint-disable-next-line complexity
-const ui = async (options, {pkg, rootDir}) => {
+const ui = async (options, {pkg, rootDir, isYarnBerry}) => {
 	const oldVersion = pkg.version;
 	const extraBaseUrls = ['gitlab.com'];
 	const repoUrl = pkg.repository && githubUrlFromGit(pkg.repository.url, {extraBaseUrls});
-	const pkgManager = options.yarn ? 'yarn' : 'npm';
+
+	function getPkgManager() {
+		if (!options.yarn) {
+			return npm;
```

Comment:
```suggestion
			return 'npm';
```

Reply:
Nice catch! This repo could really benefit a lot from typescript